config/install/og.settings.yml

```diff
@@ -1 +1,6 @@
+features_ignore_og_fields: false
```

Comment:
I also think this is not core functionality. This is definitely not an 80% case. This is more something for an `og_features` sandbox, or to be offered as a patch in the issue queue for whoever needs it.